RELEASENOTES.md

```diff
@@ -1,7 +1,960 @@
-This file is a placeholder on the primary development branch of the
-OFI NCCL Plugin so that "make dist" works properly.  Release branches
-will have an accurate release history in this location, and each
-release tarball will also have up to date release notes.
+# AWS OFI NCCL Release notes
+
+# Supported Distributions
+
+- Amazon Linux 2 and Amazon Linux 2023
+- Ubuntu 20.04 LTS, 22.04 LTS and 24.04 LTS.
+
+# v1.16.0 (2025-06)
+
+The 1.16.0 release series supports [NCCL v2.27.5-1](https://github.com/NVIDIA/nccl/releases/tag/v2.27.5-1) while maintaining backward compatibility with older NCCL versions (([NCCL v2.17.1](https://github.com/NVIDIA/nccl/releases/tag/v2.17.1-1) and later).
+
+With this release, building with platform-aws requires [Libfabric v1.22.0amzn4.0](https://github.com/aws/libfabric/commits/v1.22.0amzn4.0) or greater. And it is currently tested with versions up to [Libfabric 2.1.0amzn3](https://github.com/aws/libfabric/commits/2.1.0amzn3.0).
+
+### Bug Fixes and Improvements:
+
+- Infrastructure Changes
+  - Added functionality to allow of environment modification in a thread safe way.
+  - Added a connection management module to better match the NCCL API.
+  - Added functionality to allow communicators to abort gracefully to support the NCCL fault tolerance feature
+- Tuner Enhancements
+  - Enabled region based tuner by default for improved performance
+  - Optimized P6 tuning with specific adjustments for nChannels at mid-message sizes (4-32MB)
+  - Adjusted P6 region tuner for 0x0 32 rank configurations to use Ring algorithm for better performance with large messages
+  - Added libnccl-tuner-ofi.so symlink for easier configuration with `NCCL_TUNER_PLUGIN=ofi`
+
+### OFI NCCL plugin runtime environment variable changes:
+
+**Default values for p4d/p4de instances:**
+- `NCCL_BUFFSIZE`: 8388608
+- `NCCL_P2P_NET_CHUNKSIZE`: 524288
+
+**Default values for p-series/p5 instances:**
+- `NCCL_BUFFSIZE`: 8388608
+- `NCCL_P2P_NET_CHUNKSIZE`: 524288
+- `NCCL_NVLSTREE_MAX_CHUNKSIZE`: 524288
+- `NCCL_NVLS_CHUNKSIZE`: 524288
+- `NCCL_NET_FORCE_FLUSH`: 0
+
+**Other changes:**
+- `PROTOCOL`: "default" (Set protocol to a dummy name from NULL since it is based on system support and rail selection)
+- `NET_LATENCY`: 0.0 (Changed internode network latency to a float from an int with a default of 0.0)
+- `TUNER_TYPE`: "Region" (Set region based tuner as default from NULL)
```

Comment:
This is not a user-visible change.  Don't list it here.  We just changed how we set the defaults.